src/modules/levels/bloodbath.js

```diff
@@ -0,0 +1,32 @@
+/*
+    Level Description & Author Info here
```

Comment:
I'm glad you added auther info! Great!

Feel free to remove this line. (Line 2)